Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -112,6 +118,25 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
     cell.contentView.alpha = 0.5
     #endif
   }
+    
+  func requestLocation() {
+    if CLLocationManager.locationServicesEnabled() {
+      self.locationManager.requestLocation()
+    }
+  }
+    
+  func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+    let userLocation:CLLocation = locations[0] as CLLocation
+    CLGeocoder().reverseGeocodeLocation( userLocation) { (placemarks, error) in
```

Comment:
Need to format files. No space before `userLocation`

Reply:
Weird I ran the formatter.